devel.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get update && apt-get install -y qt5-default libqt5webkit5-dev \
 
 COPY vendor/ ./vendor
 COPY Gemfile* ./
-COPY entrypoint.sh ./
+COPY devel.entrypoint.sh ./entrypoint.sh
 
 RUN bundle -j4
 
```

Comment:
Please update the `ENTRYPOINT` on line 15.

Reply:
Sorry, nevermind! :smile: I see it's renamed when copied.

Reply:
okay :D

Reply:
This change is required. Sorry for the confusion.